src/main/steps/applicant1/interim-applications/alternative-service/why-apply-this-way/content.ts

```diff
@@ -0,0 +1,168 @@
+import { isEmpty } from 'lodash';
+
+import { AlternativeServiceDifferentWays, AlternativeServiceMethod, YesOrNo } from '../../../../../app/case/definition';
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent } from '../../../../../app/form/Form';
+import { CommonContent } from '../../../../common/common.content';
+
+const en = (
+  { partner }: CommonContent,
+  multipleWaysSelected: boolean,
+  emailSelected: boolean,
+  textMessageSelected: boolean,
+  whatsAppSelected: boolean,
+  socialMediaSelected: boolean,
+  cannotUploadEvidence: boolean
+) => ({
+  title: `${cannotUploadEvidence ? 'Why are you applying to send the papers ' : 'Why do you think sending the papers '}
+    ${
+      multipleWaysSelected
```

Comment:
I think we could remove the first ternary check `multipleWasysSelected` because the default value returned is the same as this check? 🤔